Lotte_SweetPatatos_Warr-Cinema/src/dao/MemberDao.java

```diff
@@ -87,4 +89,60 @@ public boolean save(MemberDto memberDto) {
         }
         return false;
     }
+    
+    public JSONObject findByReserve(int user_id) {
+
+		String sql = " select title,poster,cineName,runningDate,reserveStartTime,reserveEndTime, mv.id , rn.id from member m "
+				+ "	join reservation r " + "	on r.member_id = m.id " + "	join movie mv " + "	on r.movie_id = mv.id "
+				+ "	join running rn " + "	on r.running_id = rn.id " + " where m.id = ?; ";
+
+		JSONObject jsonob = new JSONObject();
+
+		try (
+          Connection conn = DBConnection.getConnection(); 
+          PreparedStatement psmt = conn.prepareStatement(sql);
+		) {
+			psmt.setInt(1, user_id);
+			ResultSet rs = psmt.executeQuery();
```

Comment:
```suggestion
try (
    ResultSet rs = psmt.executeQuery();
)
```
이 부분 코멘트를 까먹었네요. 이렇게 안하면 close 되지 않습니당~